pennylane_lightning/core/src/gates/Gates.hpp

```diff
@@ -516,14 +516,17 @@ static auto getSingleExcitation(T angle) -> std::vector<ComplexT<T>> {
             ZERO<ComplexT, T>(),
             ZERO<ComplexT, T>(),
             ZERO<ComplexT, T>(),
+
             ZERO<ComplexT, T>(),
             c,
-            s,
+            -s,
```

Comment:
Is it a bug here?

Reply:
Yes, mixed up column/row-major ordering.